dissect/target/plugins/apps/vpn/openvpn.py

```diff
@@ -166,14 +166,18 @@ def _parse_config(content: str) -> dict[str, Union[str, list[str]]]:
     """Parses Openvpn config  files"""
     lines = content.splitlines()
     res = {}
+    boolean_fields = OpenVPNServer.getfields("boolean") + OpenVPNClient.getfields("boolean")
```

Comment:
I like the idea of special casing the known boolean fields. If you turn this into a set of field names, you can more easily check in the for loop and don't need an extra inner for loop anymore, e.g.:

```suggestion
    boolean_fields = OpenVPNServer.getfields("boolean") + OpenVPNClient.getfields("boolean")
    boolean_field_names = set(boolean_field.name for boolean_field in boolean_fields)
```

```
if key in boolean_field_names:
```